tracer/src/Datadog.Trace/AppSec/EventTrackingSdkV2.cs

```diff
@@ -0,0 +1,242 @@
+// <copyright file="EventTrackingSdkV2.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Datadog.Trace.AppSec.Coordinator;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.AppSec;
+using Datadog.Trace.SourceGenerators;
+using Datadog.Trace.Tagging;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.AppSec;
+
+/// <summary>
+/// Allow
+/// </summary>
+public static class EventTrackingSdkV2
+{
+    /// <summary>
+    /// Sets the details of a successful login on the local root span
+    /// </summary>
+    /// <param name="userLogin">The userLogin associated with the login success</param>
+    /// <param name="userId">The optional userId associated with the login success</param>
+    /// <param name="metadata">The optional metadata associated with the login success</param>
+    [PublicApi]
+    public static void TrackUserLoginSuccess(string userLogin, string? userId = null, IDictionary<string, string>? metadata = null)
+    {
+        TelemetryFactory.Metrics.Record(PublicApiUsage.EventTrackingSdkV2_TrackUserLoginSuccess_UserId);
```

Comment:
I think you end up calling `TelemetryFactory.Metrics.Record(PublicApiUsage.EventTrackingSdkV2_TrackUserLoginSuccess_UserId);` twice, once here and once when the automatic instrumentation kicks in. Essentially, the Datadog.Trace.Manual should be the only public entrypoint now when we ship the `Datadog.Trace` package, so all calls will be routed through the automatic instrumentation.

Reply:
This feedback also applies to the other public methods in this class

Reply:
I wonder if you could get rid of all of these public methods and leave only internal ones 🤔

Reply:
I did thanks, they were useful if we dont reference the manual library and tests from our asm samples, which isnt the case here, fixed in  bf0b85b385f29508dc4d0406e0770d77d13b1ab0